config/routes.rb

```diff
@@ -1,3 +1,24 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get '/', to: 'welcome#index'
+  get '/merchants', to: 'merchants#index'
+  get '/items', to: 'items#index'
+  get '/merchants/:merchant_id/items', to: 'merchants#item_index'
+
+  get '/merchants/new', to: 'merchants#new'
+  get '/merchants/:merchant_id/items/new', to: 'items#new'
+
+  get '/merchants/:id', to: 'merchants#show'
+  get '/items/:id', to: 'items#show'
+
+  get 'items/:id/edit', to: 'items#edit'
+  get '/merchants/:id/edit', to: 'merchants#edit'
+
+  post '/merchants', to: 'merchants#create'
+  post '/merchants/:merchant_id/items', to:'items#create'
+
+  patch '/merchants/:id', to: 'merchants#update'
+  patch '/items/:id', to: 'items#update'
+
+  delete '/merchants/:id', to: 'merchants#delete'
```

Comment:
the restful action we'd have this go to is destroy .... "merchants#destroy" ... so your action/method in your merchants controller should also be destroy and not delete. Same thing for both of your delete in your items controller.